test/libsolidity/semanticTests/structs/struct_delete.sol

```diff
@@ -0,0 +1,13 @@
+contract C {
+    struct S { uint256 x; }
+    S s;
```

Comment:
```suggestion
    uint8 before = 23;
    S s;
    uint8 after = 17;
```
and make sure those two are still unchanged after the delete maybe?
And maybe use something other than ``uint256`` in the struct, i.e. something shorter than a full slot?
Also (but maybe in a separate test) we may want to check a struct that's larger than one slot?

Reply:
I added few more fields to this struct in this test. And more checks of course. Hope that it is fine like that.